977_squares_of_sorted_array.py

```diff
@@ -0,0 +1,10 @@
+from typing import List
+
+
+class Solution:
+    def sortedSquares(self, nums: List[int]) -> List[int]:
```

Comment:
This solution is intuitive, but you can solve this with more efficient solution.